events/tests/test_api.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+from graphql_relay import to_global_id
+
+from common.tests.utils import assert_permission_denied
+from events.factories import EventFactory, OccurrenceFactory
+
+
+@pytest.fixture(autouse=True)
+def autouse_db(db):
+    pass
+
+
+EVENTS_QUERY = """
+query Events {
+  events {
+    edges {
+      node {
+        name
+        shortDescription
+        description
+        createdAt
+        updatedAt
+        duration
+        occurrences {
+          edges {
+            node {
+              time
+              venue {
+                name
+                seatCount
+                description
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+"""
+
+EVENT_QUERY = """
+query Event($id:ID!) {
+  event(id: $id) {
+    id,
+    name,
+    shortDescription,
+    description,
+    createdAt,
+    updatedAt,
+    duration,
+    occurrences{
+      edges{
+        node{
+          time,
+          venue{
+            name,
+            seatCount,
+            description
+          }
+        }
+      }
+    }
+  }
+}
+"""
+
+OCCURRENCES_QUERY = """
+query Occcurences {
```

Comment:
Typo.

Reply:
> Good stuff! I don't think there is actually any need to restrict these queries to logged-in users only (sorry for practically non-existing specs for these queries), but I guess we can keep them like that for now, and change later if/when needed.
> 
> One thing, GraphQL queries don't normally include commas, so could you remove them from the test queries.

Yeah I think for now we can leave it like this, this will need to be updated anyway when we have more specs (filters, two types of admin ...)